examples/twittermap/web/app/views/twittermap/index.scala.html

```diff
@@ -78,9 +83,24 @@ <h1> About </h1>
     <exception-bar></exception-bar>
   </div>
 
-
-
 </div>
 
-
 }
+
+<script>
+
+  $(document).ready(function() {
```

Comment:
Put a comment about this function before line 92. What does this function do? And how does it work?